autogen.sh

```diff
@@ -47,4 +47,5 @@ autoheader
 automake -a --add-missing -Wall
 ( cd src/gmock && autoreconf -fvi; )
 ( cd src/rocksdb && autoreconf -fvi; )
+( cd src/accelio && autoreconf -fvi; )
```

Comment:
this either needs to be conditional on src/accelio existing, or we need to make sure the 'make dist' tarball includes the (possibly empty) directory.